gloo/transport/uv/libuv.h

```diff
@@ -598,26 +598,20 @@ class TCP final : public Handle<TCP, uv_tcp_t> {
 
   void read(char* ptr, size_t length) {
     reads_.emplace_back(ptr, length);
-    auto rv = uv_read_start(
-        this->template get<uv_stream_t>(), &uv__alloc_cb, &uv__read_cb);
-#ifdef _WIN32
```

Comment:
Hey @gunandrose4u, do you know if this change will break Windows support?

Reply:
The Windows job in CircleCI is passing. Is that enough?

Reply:
It works for Windows.